nextflow_schema.json

```diff
@@ -539,6 +551,25 @@
                     "default": "bam_stat,inner_distance,infer_experiment,junction_annotation,junction_saturation,read_distribution,read_duplication",
                     "fa_icon": "fas fa-chart-pie",
                     "description": "Specify the RSeQC modules to run."
+                },
+                "contaminant_screening": {
+                    "type": "string",
+                    "description": "Tool to use for detecting contaminants in unaligned reads - available options are 'kraken2' and 'kraken2_bracken'",
+                    "fa_icon": "fas fa-virus-slash",
+                    "enum": ["kraken2", "kraken2_bracken"]
+                },
+                "kraken_db": {
+                    "type": "string",
+                    "description": "Database when using Kraken2/Bracken for contaminant screening.",
+                    "help_text": "See [pre-built Kraken2/Bracken databases](https://benlangmead.github.io/aws-indexes/k2).",
```

Comment:
Never tried it, but I doubt Markdown hyperlinks in the `nextfow_schema.json` are rendered. Could also cause problems if so. I think it may be safer to include the URL as plain string?

Reply:
Decided to just refer the user to usage where it is discussed in full